sp/src/game/client/hl2/c_script_intro.cpp

```diff
@@ -474,8 +475,11 @@ void C_PlayerViewProxy::GetEyePosition( Vector &vecOrigin, QAngle &angAngles )
 const QAngle &C_PlayerViewProxy::LocalEyeAngles( void )
 {
 	C_BasePlayer *pPlayer = GetPlayer();
-	if (pPlayer)
-		return GetAbsAngles() + (pPlayer->LocalEyeAngles() - pPlayer->GetAbsAngles());
+	if (pPlayer) {
+		static QAngle angAngles;
```

Comment:
It may be worth turning these statics into member variables for when Mapbase begins running on MP and a player-by-player distinction may be necessary, but this is fine for now.